mentat/conversation.py

```diff
@@ -151,6 +151,7 @@ async def get_model_response(self) -> list[FileEdit]:
         tokens = count_tokens(conversation_history, self.model)
         response_buffer = 1000
         code_message = await code_context.get_code_message(
+            messages[-1]["content"],
```

Comment:
This might not necessarily always be the user's input; it might have some extra text (check out code_edit_feedback.py), or it might just not be the user's input (it won't be soon with a PR I have coming out soon), not to mention the most recent feedback might have just been 'you made a mistake like this, fix it', which won't give us very good embeddings. I think it might be best to just pass all of the user's actual inputs; in general, the user's inputs aren't very long, so I think this could work pretty well. In an upcoming PR I automatically store all of the user's actual inputs (as opposed to a few things we give the role of user but are given by us) in a list, so we could use that then if you think that's a good idea

Reply:
I see, ya that makes sense. Shall we leave it as-is for this PR and update it when that other PR is merged?

Reply:
Hmm it does get complex when the user has sent a second message / request. The embedding of just the second message might totally miss context that's in the first message. So the system would appear to get a lot dumber on the second prompt

Reply:
let's not worry about it for now, as our goal here is getting embeddings added. Perhaps the right way to do it will be have gpt-3 create things to embed and search for, relevant to the last prompt, given the conversation history as well